Source/Scene/Cesium3DTileset.js

```diff
@@ -787,6 +799,7 @@ define([
         var length = children.length;
         for (var i = 0; i < length; ++i) {
             var child = children[i];
+            child.updateTransform(); // Need to update transform and bounding volumes before checking distance
```

Comment:
I decided to decouple it. The transform update needs to happen always, but we may not always check the distance from camera (like the recent PERFORMANCE_IDEA about temporal coherence).